src/app/src/main/kotlin/org/vaccineimpact/api/app/app_start/Endpoint.kt

```diff
@@ -107,4 +107,9 @@ fun Endpoint.basicAuth(): Endpoint
     return this.copy(basicAuth = true)
 }
 
+fun Endpoint.post(): Endpoint
+{
+    return this.copy(method = spark.route.HttpMethod.post)
+}
```

Comment:
Nice. I'm really liking this fluent API for building endpoints, feels so easy to use!